apps/web/src/components/tweet/Timeline.tsx

```diff
@@ -105,8 +110,8 @@ export const Timeline = ({
       </Text>
 
       {refresh ? <Spinner position="absolute" top="15%" right="50%" /> : null}
-
-      {renderTweets()}
+      <TweetSearch />
+      {/* {renderTweets()} */}
```

Comment:
code smell? It is not a good idea to keep the unused code commented, you can recover the commit history anytime